indigo_app/forms.py

```diff
@@ -276,6 +300,104 @@ def filter_queryset(self, queryset):
         if self.cleaned_data.get('taxonomies'):
             queryset = queryset.filter(taxonomies__in=self.cleaned_data.get('taxonomies'))
 
+        # Advanced filters
+        # filter by assent date range
+        if self.cleaned_data.get('assent_date_check') and self.cleaned_data['assent_date_check']:
```

Comment:
You shouldn't need the second expression. `if self.cleaned_data.get('assent_date_check')` should be sufficient.

Reply:
This repeated block of code for the date checker cries out to be turned into a helper function :)

Reply:
Let me look into it